mysite/chat/api.py

```diff
@@ -241,6 +242,8 @@ def get(self, request, enroll_key, chat_id, *args, **kwargs):
             view.request = self.request
             chat, _ = view.get_or_init_chat(enroll_code, chat_id)
             return Response(ChatSerializer(chat).data)
+        else:
+            raise Http404() 
```

Comment:
currently no, but we need it ( maybe not in this branch, but it's only 2 lines of code).